keygenerator.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
This file, along with `scurl.sh` are installed in your path as part of the `ccf` Python package.